charts/pg-db/templates/cluster.yaml

```diff
@@ -13,173 +13,81 @@ metadata:
+    {{- end}}
+    {{- if .Values.customReplicationTLSSecret.name }}
+    customReplicationTLSSecret:
+      name: .Values.customReplicationTLSSecret.name
+    {{- end }}
   {{- end }}
-  userLabels:
-    pgo-version: {{ .Chart.AppVersion }}
-  pgPrimary:
-    image: {{ include "pg-database.primary-image" . }}
-    resources:
-      requests:
-        memory: {{ .Values.pgPrimary.resources.requests.memory }}
-    tolerations: {{ .Values.pgPrimary.tolerations }}
-    volumeSpec:
-      size: {{ .Values.pgPrimary.volumeSpec.size }}
-      accessmode: {{ .Values.pgPrimary.volumeSpec.accessmode }}
-      storagetype: {{ .Values.pgPrimary.volumeSpec.storagetype }}
-      storageclass: {{ .Values.pgPrimary.volumeSpec.storageclass }}
-    expose:
-      serviceType: {{ .Values.pgPrimary.expose.serviceType }}
-      {{- if eq .Values.pgPrimary.expose.serviceType "LoadBalancer" }}
-      {{- if .Values.pgPrimary.expose.loadBalancerSourceRanges }}
-      loadBalancerSourceRanges:
-{{ toYaml .Values.pgPrimary.expose.loadBalancerSourceRanges | indent 6  }}
-      {{- end }}
-      {{- if .Values.pgPrimary.expose.loadBalancerIP }}
-      loadBalancerIP: {{ .Values.pgPrimary.expose.loadBalancerIP }}
-      {{- end  }}
-      annotations:
-        pg-cluster-annot: {{ include "pg-database.fullname" . }}
-      labels:
-        pg-cluster-label: {{ include "pg-database.fullname" . }}
-      {{- end }}
-    affinity:
-      antiAffinityType: {{ .Values.pgPrimary.affinity.antiAffinityType}}
-      {{- if .Values.pgPrimary.affinity.nodeAffinityType }}
-      nodeAffinityType: {{ .Values.pgPrimary.affinity.nodeAffinityType }}
-      {{- end }}
-      {{- if .Values.pgPrimary.affinity.nodeLabel }}
-      nodeLabel:
-{{ toYaml .Values.pgPrimary.affinity.nodeLabel | indent 8 }}
-      {{- end }}
-      {{- if .Values.pgPrimary.affinity.advanced }}
-      advanced:
-{{ toYaml .Values.pgPrimary.affinity.advanced | indent 8 }}
+  instances:
+    - name: instance1
+      replicas: 3
+      dataVolumeClaimSpec:
+        accessModes:
+        - ReadWriteOnce
+        resources:
+          requests:
+            storage: 5Gi
+  proxy:
+    pgBouncer:
+      image: {{ include "pg-database.pgbouncer-image" .}}
+      replicas: {{ .Values.proxy.pgBouncer.replicas}}
+      exposeSuperusers: {{ .Values.proxy.pgBouncer.exposeSuperusers | default true }}
+      {{- if .Values.proxy.pgBouncer.resources }}
+      resources:
+        {{- if .Values.proxy.pgBouncer.resources.requests }}
+        requests:
+          cpu: {{ .Values.proxy.pgBouncer.resources.requests.cpu }}
+          memory: {{ .Values.proxy.pgBouncer.resources.requests.memory }}
+        {{- end}}
+        {{- if .Values.proxy.pgBouncer.resources.limits }}
+        limits:
+          cpu: {{ .Values.proxy.pgBouncer.resources.limits.cpu }}
+          memory: {{ .Values.proxy.pgBouncer.resources.limits.memory }}
+        {{- end}}
+      {{- end}}
+      {{- if .Values.proxy.pgBouncer.expose }}
+      expose:
```

Comment:
We seem to be missing fields which we have in values like: proxy.pgbouncer..sidecars, proxy.pgbouncer.tolerations, proxy.pgbouncer.topologySpreadConstraints, proxy.pgbouncer.config, proxy.pgbouncer.customTLSSecret and some others probably.